Fix/15517261 mobile wallet menu

KatarzynaStefanowiczPagepro opened the pull request:
## Description

- [] TODO write description

**New stuff** ✨

* New stuff goes here

**Changes** 🏗

* Changes go here

**Deletions** ⚰️

* Deletions go here

## TODO

- [ ] Add TODOs

(Resolves | Contributes to) #31415

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/JoinColony/colonyCDapp?pullRequest=882) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/JoinColony/colonyCDapp?pullRequest=882) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/JoinColony/colonyCDapp?pullRequest=882) it.</sub>

KatarzynaStefanowiczPagepro commented:
https://tw.auroracreation.com/app/tasks/15517261

Mobile wallet menu items
- added action to user hub button inside the menus
- cleaned code

![image](https://github.com/JoinColony/colonyCDapp/assets/119701775/1f506cf6-b952-4a82-9c2a-f606c228bb57)

CzarekDryl commented on src/components/common/Extensions/UserNavigation/partials/UserMenu/UserMenu.tsx:
```diff
@@ -125,27 +128,29 @@ const UserMenu: FC<UserMenuProps> = ({
       </div>
       <div
-        className={clsx('transition-transform', {
+        className={clsx('transition-transform rm pt-[4.1625rem] sm:pt-0', {
```
What's `rm`?

KatarzynaStefanowiczPagepro replied:
Fixed
